### PR TITLE
Maxsibilla/issue 158

### PR DESCRIPTION
**Pull request by maxsibilla**

### CODE REVIEW

**Comment by maxsibilla** [comment]
This PR assumes that you are using Node.js 20 and updates Next.js from 13 -> 14. There is a slight issue with the package `antd` so I had to transpile that package as well as the other packages that it utilizes. See `next.config.js` for these changes.

To get Node.js version 20:
```
npm install n -g
n stable
```

**Comment by DerekFurstPitt** [comment]
Looks good on my end.

**Review by yuanzhou — APPROVED** [review]
@maxsibilla thanks for the upgrade. I tested on my local with nodejs v20.11.0 and all good. Will merge and redeploy on DEV once @libpitt also approves.

**Pull request merged**
